packages/internal/src/cliLogger.ts

```diff
@@ -0,0 +1,20 @@
+import { defaultLoggerOptions } from '@redwoodjs/api/src/logger'
+
+// Can't use color in the createLogger logger - so use a simpler set of log fns
+const logLevel = defaultLoggerOptions.level
+type CLog = (typeof console)['log']
+
+/**
+ * An alternative to createLogger which supports the same logging levels
+ * but allows for full ANSI when printing to the console.
+ */
+export const cliLogger: CLog & {
+  trace: CLog
+  debug: CLog
+} = function (...data: any[]) {
+  console.log(data)
```

Comment:
Did you mean to spread this here instead of passing it through? Right now this shows up in my terminal as arrays, which does have a certain style to it but wasn't sure if it was intentional:

```suggestion
  console.log(...data)
```

```
gen | [ 'Generating full TypeScript definitions and GraphQL schemas' ]
gen | [ 'Done.' ]
```